streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -373,27 +373,51 @@ abstract class DStream[T: ClassTag] (
   /**
    * Wrap a body of code such that the call site and operation scope
    * information are passed to the RDDs created in this body properly.
-   */
-  protected def createRDDWithLocalProperties[U](time: Time)(body: => U): U = {
+   * @param body RDD creation code to execute with certain local properties.
+   * @param time Current batch time that should be embedded in the scope names
+   * @param displayInnerRDDOps Whether the detailed callsites and scopes of the inner RDDs generated
+   *                         by `body` will be displayed in the UI; only the scope and callsite will
+   *                         be of `this` DStream will be displayed.
+   */
+  protected[streaming] def createRDDWithLocalProperties[U](
+      time: Time, displayInnerRDDOps: Boolean)(body: => U): U = {
```

Comment:
I find that look more odd. one short line followed by a long line, and the curried function is also on that line .... harder to read IMO.